dev-docs/source/OSXCondaPackaging.rst

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
Thanks for adding this. It will be very useful for introducing those new to this whole packaging saga.